CHANGELOG.md

```diff
@@ -3,6 +3,10 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [1.2.0] - TBD
+### Added
+- Old locations for Table and Partitions along with Housekeeping-as-a-service(HKAAS) table parameters to ApiarySNSListener
```

Comment:
make more generic: maybe two changes one for:
`Old locations for Table and Partitions are add tot the ApiarySNSListener message`
`Configurable regex to control which table parameters will be added to the ApiarySNSListener message, default non`

Reply:
Yes please, changelog should have one line per change and this PR contains 2 (arguably we should have split it into two PRs).

Reply:
done.